apps/meteor/server/startup/migrations/v283.ts

```diff
@@ -0,0 +1,21 @@
+import { Settings } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+import { Users } from '../../../app/models/server';
+
+addMigration({
+	version: 283,
+	up() {
+		Users.update(
```

Comment:
I'm not sure if it is worth to have a migration for the whole userbase (open has about 300k users for example). @sampaiodiego wdyt?

Reply:
I'd prefer to cleanup the data. But I agree it may be heavy since we don't have an index for this field.